README.md

```diff
@@ -10,8 +10,6 @@ This is a **fork** of the MultiMC Launcher and not endorsed by MultiMC. The Poly
 <br>
 
 # Installation
-- All packages (archived by version) can be found [here](https://packages.polymc.org/) ([latest](https://packages.polymc.org/latest)).
```

Comment:
why

Reply:
Because we are moving to GitHub releases, but I agree it should probably stay until that’s done

Reply:
keep in mind jenkins contains generic linux builds that GHA doesn't have, and I'm going to implement BSD builds as well

Reply:
We build on Ubuntu 20.04 and produce a portable zip. Might also add system builds